frontend/src/app/app/quests/socials/layout.tsx

```diff
@@ -7,7 +7,7 @@ import NavMain from './components/NavMain'
 import { ReactNode, useContext } from "react";
 import { usePathname } from 'next/navigation'
```

Comment:
why do you need a layout file? look at the structure for the profile/page.tsx and see how the sidebar was implemented there.